src/Streamly/Internal/System/Process.hs

```diff
@@ -135,7 +135,7 @@ import Streamly.Internal.System.IO (defaultChunkSize)
 import qualified Streamly.Internal.Console.Stdio as Stdio
 import qualified Streamly.Internal.Data.Stream.Chunked
     as ArrayStream (arraysOf)
-import qualified Streamly.Data.Stream.Concurrent as Concur
+import qualified Streamly.Data.Stream.Prelude as Concur
```

Comment:
This can be imported as "Stream" and "import qualified Streamly.Data.Stream as Stream" can be removed.